eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -129,13 +129,12 @@ author: $author
 ms.author: $msauthor
 ms.date: $date
 ms.topic: reference
```

Comment:
Why were these removed?

Reply:
This is according the their metadata guidance. If we use the ms.service, then the set of `ms.prod` and `ms.technology` are not necessary. 
https://raw.githubusercontent.com/Azure/azure-docs-sdk-java/main/docs-ref-services/preview/sdk-template-readme.md

Reply:
There are many OPS warning regarding we are using wrong ms.prod value